workLoggingSystem.html

```diff
@@ -98,42 +98,40 @@
 
                     <!-- <div class="row jobTicket">
                         <div class="col-md-4 jobTicketCard" id="y" onclick="showDetails('y')">
-                            <div class="row frameHead">
-                                <div class="col-md-3">
                                     <div class="row">
-                                        <div class="col menu">Id:</div>
-                                        <div class="col frameId">B0001</div>
+                                        <div class="col-md-12">
+                                         <div class="row frameHead">
```

Comment:
Please close the div, Please Use some html formatter to detect such basic issue

Reply:
The div was already closed according to the code. (it was closed at last)

Reply:
Okay cool, Make sure to show me before pushing the changes